shared/common-adapters/text.desktop.js

```diff
@@ -55,6 +55,9 @@ class Text extends Component<Props> {
   }
 
   render() {
+    if (!this.props.type) {
```

Comment:
if you forget to set this, flow will eventually catch it but while debugging it'll give you an obscure error about the fontSize being missing so lets just make this error more explicit